typescript/sdk/src/token/deploy.ts

```diff
@@ -58,18 +58,21 @@ abstract class TokenDeployer<
   }
 
   async constructorArgs(_: ChainName, config: TokenRouterConfig): Promise<any> {
+    // TODO: derive from decimal ratios?
```

Comment:
Can you turn this into a Github issue and link it? I would also agree that we should do this based on decimal ratios, and not add another config option #configandartifacts

Reply:
But it sounds like we already had this?